trpcultivate_phenotypes/src/Plugin/TripalImporter/TripalCultivatePhenotypesTraitsImporter.php

```diff
@@ -1472,24 +1472,45 @@ public function postRun() {
   }
 
   /**
-   * {@inheritdoc}
+   * Provide a more informative description about the template file.
```

Comment:
This feels more narrow then this method actually is. What about

```suggestion
   * Describe the upload format including column descriptions + template file.
```